app/assets/javascripts/charts.js

```diff
@@ -320,41 +317,6 @@ function bind_hover_event(target, formater) {
   });
 }
 
-function search_on_click(event, item) {
```

Comment:
This makes me :panda_face::cry: but we have to keep this function, if we want to maintain full functionality of `flot_pie_chart()`

Reply:
Or do we? Not after #8366